src/js/components/FilmItem/filmItem.test.js

```diff
@@ -21,7 +22,9 @@ describe('FilmItem', () => {
   });
   it('should render correctly', () => {
     const wrapper = render(
-      <FilmItem {...props} /> );
+      <MemoryRouter>
```

Comment:
What do you think about mocking `react-router`s components? Since we don't really need those rendered in snapshots.